config/marc_liberation.yml

```diff
@@ -7,31 +7,31 @@ recap_dump_records_per_file: 5000
 test_records_per_file: 25
 
 dump_types:
-- label: 'Changed Records'
-  constant: 'CHANGED_RECORDS'
-- label: 'All Bib IDs'
-  constant: 'BIB_IDS'
-- label: 'All Bib IDs with Holdings'
-  constant: 'MERGED_IDS'
-- label: 'All Records'
+- label: 'All Records' # used for full dumps
   constant: 'ALL_RECORDS'
-- label: 'Updated Princeton ReCAP Records'
+- label: 'Changed Records' # used for incremental dumps
+  constant: 'CHANGED_RECORDS'
+- label: 'Updated Princeton ReCAP Records' # used for records we send to SCSB
   constant: 'PRINCETON_RECAP'
-- label: 'Updated Partner ReCAP Records'
+- label: 'Updated Partner ReCAP Records' # used for records we get from SCSB partners
   constant: 'PARTNER_RECAP'
```

Comment:
Can we also update to: used for records we get from SCSB partners (NYPL and Columbia)